WcaOnRails/lib/database_dumper.rb

```diff
@@ -350,6 +350,54 @@ def self.actions_to_column_sanitizers(columns_by_action)
         ),
       ),
     }.freeze,
+    "competition_venues" => {
+      where_clause: JOIN_WHERE_VISIBLE_COMP,
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w(
+          id
+          competition_id
+          wcif_id
+          name
+          latitude_microdegrees
+          longitude_microdegrees
+          timezone_id
+          created_at
+          updated_at
+        ),
+      ),
+    }.freeze,
+    "venue_rooms" => {
+      where_clause: "JOIN competition_venues ON competition_venues.id = competition_venue_id #{JOIN_WHERE_VISIBLE_COMP}",
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w(
+          id
+          competition_venue_id
+          wcif_id
+          name
+          created_at
+          updated_at
+        ),
+      ),
+    }.freeze,
+    "schedule_activities" => {
+      #FIXME: this does not handle nested activities, which will be omitted!
+      where_clause: "JOIN venue_rooms ON (venue_rooms.id = holder_id AND holder_type = \"VenueRoom\") JOIN competition_venues ON competition_venues.id = competition_venue_id #{JOIN_WHERE_VISIBLE_COMP}",
```

Comment:
I didn't quite manage to figure out a query to get `schedule_activities` that are nested within `schedule_activities` that are nested within `VenueRoom`s for a visible competition.
There could be an arbitrary depth for nested activity, so I'm not quite sure how to tackle the issue.
So far this should be enough.